server/games/game.py

```diff
@@ -80,6 +81,8 @@ def __init__(
         self.game_service = game_service
         self._player_options: Dict[int, Dict[str, Any]] = defaultdict(dict)
         self.launched_at = None
+        self.startTime = None
+        self.endTime = None
```

Comment:
Variable names should be snake_case

Reply:
I know the table columns aren’t, but that’s just cuz the database schema is a bit of a mess

Reply:
do i have to use hardcoded timestemps for the tests ?

Reply:
You should probably mock the calls to `datetime.now` to return a specific value and then assert that the value shows up in the database.